README.md

```diff
@@ -72,10 +81,28 @@ Write one Express route for each of the four HTTP methods.
 Then, make each route respond with a one-word string containing the RESTful action that would most likely be associated with this route.
 
 ```js
+// had to check last nights homework for assistance with this one.
 var express = require("express");
 var app = express();
 
```

Comment:
if you use ES6 bindings over ES5 bindings, then remove the ES5 bindings. don't mix.